pkg/scheduler/testing/wrappers.go

```diff
@@ -255,6 +255,12 @@ func (p *PodWrapper) Priority(val int32) *PodWrapper {
 	return p
 }
 
+// PriorityClassName sets a priorityclass name into PodSpec of the inner pod.
+func (p *PodWrapper) PriorityClassName(val string) *PodWrapper {
```

Comment:
do we have a test using it?

Reply:
No, the current wrappers don't have tests yet.  We'll need a testing file for all the wrapper functions in this package. /cc @Huang-Wei

Reply:
But the priority class name is irrelevant for those unit tests (so I wouldn't change those tests); priority class name is relevant only if the test involves the api-server (integration or e2e tests) because it is the one that looks up the actual priority and sets it on pod creation.

Reply:
maybe we can wait for a valid case and then add the test utility.